usm_wrappers/salt_wrapper.py

```diff
@@ -63,6 +73,53 @@ def accept_minion(minion_id):
     return (True if out else False)
 
 
+def get_started_minions(minions=[], timeout=60):
+    minion_set = set(minions)
+    up_minions = set(runner.cmd('manage.up', []))
+    if minion_set:
+        started_minions = up_minions.intersection(minion_set)
+    else:
+        started_minions = up_minions
+
+    if not timeout:
+        return started_minions
+
+    time_spent = 0
+    while True:
+        wait = timeout - time_spent
+        if wait <= 0:
+            break
+        start_time = time.time()
+
+        ret = sevent.get_event(wait=wait, full=True, tag='salt/minion')
```

Comment:
seems strange to wait for wait time. How will you ever get more than one or two iterations?
I expect this to be a constant period that is some fraction of timeout.